templates/files/magento2/.eslintrc.json

```diff
@@ -2,40 +2,140 @@
   "env": {
```

Comment:
Should this file be symlinked/copied from https://github.com/YouweGit/coding-standard/blob/46d595616696d915374f6aaa9189ad5fc9aeec65/src/Youwe/eslintrc.json (or a Magento2-specific version if introduced into https://github.com/YouweGit/coding-standard-magento2) instead of being duplicated in this repository?

Also, keeping a copy of this file in each repository seems incorrect, as we should be adhering to the official coding standard, not a local variant of the same. Perhaps fixing this is out of scope for this pull request.